examples/list-files-and-folders/Ballerina.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+org = "wso2"
+name = "list_files_and_folders"
+version = "0.1.0"
+distribution = "2201.10.2"
+
+[[dependency]]
+org = "ballerinax"
+name = "dropbox"
+version = "0.1.0"
```

Comment:
No need to hardcode the dependency in here. We use this only for testing purposes
```suggestion
```